src/node_builtins.cc

```diff
@@ -90,15 +90,23 @@ void BuiltinLoader::GetNatives(Local<Name> property,
   auto source = env->builtin_loader()->source_.read();
   for (auto const& x : *source) {
     Local<String> key = OneByteString(isolate, x.first);
-    if (out->Set(context, key, x.second.ToStringChecked(isolate)).IsNothing()) {
+    Local<String> value;
+    if (!x.second.ToString(isolate).ToLocal(&value)) {
+      return;
+    }
+    if (out->Set(context, key, value).IsNothing()) {
       return;
     }
   }
   info.GetReturnValue().Set(out);
 }
 
 Local<String> BuiltinLoader::GetConfigString(Isolate* isolate) {
-  return config_.ToStringChecked(isolate);
+  Local<String> config_str;
+  if (!config_.ToString(isolate).ToLocal(&config_str)) {
+    return {};
+  }
+  return config_str;
```

Comment:
Let's not do this – this is dangerous and strictly worse than the previous code, because a return type of `Local<String>` (as opposed to `MaybeLocal<String>`) typically means that the value won't be empty, so the caller might assume that it's always safe to dereference the handle.